pyartcd/pyartcd/jenkins.py

```diff
@@ -118,6 +139,21 @@ def set_build_description(build: Build, description: str):
     )
 
 
+def is_build_running(build_path: str) -> bool:
+    init_jenkins()
+
+    response = requests.get(f'{constants.JENKINS_SERVER_URL}/{build_path}/api/json')
+    if response.status_code != 200:
+        logger.error('Could not fetch data for build %s', build_path)
+        raise ValueError
+
+    build_data = response.json()
+    logger.info('Build %s %s in progress',
+                f'{constants.JENKINS_UI_URL}/{build_path}',
+                'is' if build_data['inProgress'] else 'is not')
+    return build_data['inProgress']
```

Comment:
Is InProgress the only "job is still alive/running" state? I'm curious if there can be other weird but valid states that our jobs can fall in

Reply:
Ah missed that it's an attribute. Should be fine then